.env.example

```
@@ -7,6 +7,16 @@
 #          cricket, floorball, football, handball, netball
 SPORTSPRESS_SPORT=ice-hockey
 
+# WordPress site settings
+WORDPRESS_TITLE=SportsPress Test Site
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)